File: Modul 1 Sem 1/C/PRAK108-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>
#define _USE_MATH_DEFINES
#include <math.h>

int main()
{
    int rotations = 5;
    float circumference, radius, distance = 14; 

    printf("Diketahui:\n");
    printf("Pak Dengklek mengelilingi taman = %d putaran\n", rotations);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n", distance);
    printf("\n");

    circumference = distance / rotations;
    radius = circumference / (2 * M_PI);

    printf("Jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer", radius);

    return 0;
}

File: Modul 1 Sem 1/Phython/PRAK108-2410817110005-MuhammadRizqanKarim.py
from math import pi

rotations = 5
distance = 14

print("Diketahui :")
print(f"Pak Dengklek mengelilingi taman = {rotations} putaran")
print(f"Jarak tempuk Pak Dengklek {distance} Kilometer")

circumference = distance / rotations
radius = circumference / (2 * pi)

print("")
print("Jawaban :")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {radius:.2f}")
